conmon-rs/server/src/config.rs

```diff
@@ -33,6 +33,11 @@ pub struct Config {
     /// Show version information, specify "full" for verbose output.
     version: Option<Verbosity>,
 
+    #[get = "pub"]
+    #[clap(long("pause"), short('p'), value_name("PATH"))]
+    /// Run the pause process instead of the server.
+    pause: Option<PathBuf>,
```

Comment:
Sorta thinking this functionality should be a subcommand of conmonrs... ie: `conmonrs pause --path=...`. Thoughts?

Reply:
Yep, done :+1: